addons/medical_gui/stringtable.xml

```diff
@@ -799,9 +811,11 @@
         </Key>
         <Key ID="STR_ACE_Medical_GUI_Lost_Blood3">
             <English>Lost a large amount of blood</English>
+            <German>Hat eine sehr große Menge Blut verloren</German>
```

Comment:
~sehr~

Reply:
its "sehr" because in Blood2 its already "große Menge"

Reply:
Oops. Didn't see